privacyidea/lib/policy.py

```diff
@@ -2117,6 +2118,14 @@ def get_static_policy_definitions(scope=None):
                 'type': 'int',
                 'desc': _('Limit the number of allowed tokens in a realm.'),
                 'group': GROUP.TOKEN},
+            ACTION.REQUIRE_DESCRIPTION: {
+                'type': 'str',
+                'desc': _('During the rollout process, this policy makes the '
+                          'description required for all selected token-types.'),
+                'group': GROUP.ENROLLMENT,
+                'multiple': True,
+                'value': list(get_token_types())},
```

Comment:
`get_token_types()` already returns a list.

Reply:
You are right!

I accidentally wrapped get_token_types() in square brackets, which is why it didn't work for me without list().

fixed in f7ace1114f81c902ec128e78557a3554fb2001c6.